pages/index.js

```diff
@@ -67,6 +87,22 @@ export default function HomePage() {
 
   return (
     <StyledDiv>
+      <button /* Only for testing of success-marker for past days,
+        it creates the saved daily entry for yesterday.
+        Clicking with default local storage values will mark yesterday either
+        red or green depending on the currently displayed calorie count.
+        The history with entries is also logged in the console.*/
+        onClick={() => {
+          const today = new Date();
+          const day = today.getDate() - 1;
+          const month = today.getMonth();
+          const year = today.getFullYear();
+          addHistoryEntry(year, month, day, isGoalExceeded);
+          console.log("history: ", history);
+        }}
+      >
+        history test button
```

Comment:
I don't understand why you would like to have a button `history test button` in your main branch. Isn't there be a better name?